util/audio.py

```diff
@@ -15,90 +15,106 @@
 DEFAULT_WIDTH = 2
 DEFAULT_FORMAT = (DEFAULT_RATE, DEFAULT_CHANNELS, DEFAULT_WIDTH)
 
-AUDIO_TYPE_NP = 'np'
-AUDIO_TYPE_PCM = 'pcm'
-AUDIO_FILE_PREFIX = 'audio/'
-AUDIO_TYPE_WAV = AUDIO_FILE_PREFIX + 'wav'
-AUDIO_TYPE_OPUS = AUDIO_FILE_PREFIX + 'opus'
-LOADABLE_FILE_FORMATS = [AUDIO_TYPE_WAV, AUDIO_TYPE_OPUS]
+AUDIO_TYPE_NP = 'application/x-np'
+AUDIO_TYPE_PCM = 'application/x-pcm'
+AUDIO_TYPE_WAV = 'audio/wav'
+AUDIO_TYPE_OPUS = 'application/x-opus'
+SERIALIZABLE_AUDIO_TYPES = [AUDIO_TYPE_WAV, AUDIO_TYPE_OPUS]
 
 OPUS_PCM_LEN_SIZE = 4
 OPUS_RATE_SIZE = 4
 OPUS_CHANNELS_SIZE = 1
 OPUS_WIDTH_SIZE = 1
 OPUS_CHUNK_LEN_SIZE = 2
 
-NP_TYPE_LOOKUP = [None, np.int8, np.int16, None, np.int32]
-UNSUPPORTED_TYPE = 'Unsupported audio type: {}'
-
 
 class Sample:
```

Comment:
Maybe name this `AudioSample` and then `CollectionSample` can be named `TrainingSample` or `TrainingExample`? As it is the names don't tell you right away what the distinction is.

Reply:
`TrainingSample` was also my first guess, but can also be used for trst/dev. What do you think about `LabeledSample`, `Utterance` or `TranscribedSample`?

Reply:
I like `LabeledSample` and `TranscribedSample`.